docs/reference/src/language/builtins/elements.md

```diff
@@ -253,18 +253,48 @@ export component Example inherits Window {
 
 ## `GridLayout`
 
-`GridLayout` places its children in a grid. `GridLayout` adds properties to each child: `col`, `row`, `colspan`, `rowspan`.
-These properties must be compile time constants.
-You can control the position of children with `col` and `row`.
-If `col` or `row` aren't specified, they are automatically computed such that the item is next to the previous item, in the same row.
-Alternatively, the item can be put in a `Row` element.
+`GridLayout` places elements on a grid.
+
+It adds the following properties to each cell element: `col`, `row`, `colspan`, `rowspan`.
```

Comment:
On second thought, the "it adds" is a bit weird. How about this?

```suggestion
Cell elements inside a `GridLayout` obtain the following new properties:

 -   * **row** (_in_ _int_): The element's row within the grid. Set this property to override the sequential row assignment. Setting this property resets the element's column to zero, unless explicitly set.
-   * **col** (_in_ _int_): The element's column within the grid. Set this property to override the sequential column assignment.
-   * **colspan** (_in_ _int_): The number of columns this element should span. (default value: `1`)
-   * **rowspan** (_in_ _int_): The number of row this element should span. (default value: `1`)
 ```

This then also replaces the more verbose text further down.

Reply:
> Set this property to override the sequential row assignment.

You only have sequential row assignment with `Row`, right? Testing this in SlintPad, you can override the `Row`-assigned row indices with `row`, but this seems like an outlandish use case and should perhaps become an error.